### PR TITLE
upd (event): 8 June announce

### PR DESCRIPTION
**Pull request by matochu**

### CODE REVIEW

File: _events/2017-05-08_en.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+upcoming: true
+location: wix
+regLink: 'https://goo.gl/forms/joVim3ZcJnWCE4Gm2'
+regAnnounce: 'Registration on Monday, 5th of June @ 2 p.m.'
+speakers:
+  - yarosh
+  - stoyanov
+topics:
+  yarosh:
+    name: "To Orm or not to Orm, Is there a life without it ?"
```
Comment (tyv) [inline _events/2017-05-08_en.yaml]: extra space before  `?`

**Review by tyv — CHANGES_REQUESTED** [review]

File: _events/2017-05-08_en.yaml (hunk 1 above)
Comment (tyv) [reply _events/2017-05-08_en.yaml]: same in ru

Comment (matochu) [reply _events/2017-05-08_en.yaml]: I can fix it together with the following commit with additional information on one of the speakers

File: _events/2017-05-08_uk.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+sortDate: 20170608
+title: 8 червня 2017
+lang: uk
+time: '19:00'
+upcoming: true
+location: wix
+regLink: 'https://goo.gl/forms/joVim3ZcJnWCE4Gm2'
+regAnnounce: 'Pеєстрацій в понеділок, 5 червня о 14:00.'
```
Comment (AMashoshyna) [inline _events/2017-05-08_uk.yaml]: Typo : "Pеєстрація"

File: _events/2017-05-08_uk.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+speakers:
+  - yarosh
+  - stoyanov
+topics:
+  yarosh:
+    name: "To Orm or not to Orm, Is there a life without it ?"
+    lang: 'en'
+  stoyanov:
+    name: "Principles of functional programming in javascript"
```
Comment (AMashoshyna) [inline _events/2017-05-08_uk.yaml]: Capitalize title:
"Principles of Functional Programming in JavaScript"

File: _events/2017-05-08_uk.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+sortDate: 20170608
+title: 8 червня 2017
+lang: uk
+time: '19:00'
+upcoming: true
+location: wix
+regLink: 'https://goo.gl/forms/joVim3ZcJnWCE4Gm2'
+regAnnounce: 'Pеєстрація в понеділок, 5 червня о 14:00.'
```
Comment (tyv) [inline _events/2017-05-08_uk.yaml:9]: 'Pеєстрація **у** понеділок'? @AMashoshyna what u think?

Comment (AMashoshyna) [reply _events/2017-05-08_uk.yaml:9]: @tyv I'm no expert, but I googled a bit, and _"в"_ should be ok after a vowel.

**Pull request merged**